plugin.rb

```diff
@@ -32,6 +32,41 @@ def handle_post_created_edited(post, action)
     end
 end
 
+def handle_user_promoted(user_id, new_trust_level, old_trust_level)
+  name = DiscourseAutomation::Triggerable::USER_PROMOTED
+  user = User.find_by(id: user_id)
+  return if user.blank?
+
+  # don't want to do anything if the user is demoted. this should probably
+  # be a separate event in core
+  return if new_trust_level < old_trust_level
```

Comment:
I really wasn't sure whether we would want the demotions to call the trigger. I just think it's a bit weird to say you are triggering on trust level promotion, but the trigger also happens on demotion. Thoughts?